presto-hive/src/main/java/com/facebook/presto/hive/BackgroundHiveSplitLoader.java

```diff
@@ -315,6 +316,7 @@ private void invokeNoMoreSplitsIfNecessary()
                         inputFormat,
                         pathDomain,
                         isForceLocalScheduling(session),
```

Comment:
We don't need this anymore. We don't need config `isForceLocalScheduling` also. `isForceLocalScheduling` will be the same as `getNodeSelectionStrategy(session) == HARD_AFFINITY`

Reply:
Yes, I plan to remove it in the next PR